src/components/NewMessage/NewMessage.vue

```diff
@@ -706,6 +706,11 @@ export default {
 			const possibleMentions = response.data.ocs.data
 
 			possibleMentions.forEach(possibleMention => {
+				// TODO fix backend for userMention
```

Comment:
We can not, because other clients rely on the naming,

Reply:
Then I'll guess it will be better to patch response on web-client only, rather then provide both `label` and `name`?